pkg/util/deployment/deployment.go

```diff
@@ -123,6 +123,14 @@ func GetNewRCTemplate(deployment extensions.Deployment) api.PodTemplateSpec {
 	return newRCTemplate
 }
 
+// GetRCsForDeployment returns all the replication controllers for the provided deployment.
+func GetRCsForDeployment(c client.Interface, deployment extensions.Deployment) ([]api.ReplicationController, error) {
```

Comment:
Right, but since the rc selector (which is the same as the deployment selector) is propagated in the rc labels via defaulting, is this really a problem? Users shouldn't mess with rcs owned by deployments anyway.

Relevant issue:  https://github.com/kubernetes/kubernetes/issues/17491